internal/resources/common/objectmeta_schema.go

```diff
@@ -157,3 +157,14 @@ func GetTypeMapData(data map[string]interface{}) map[string]string {
 
 	return convertedMapData
 }
+
+func GetTypeIntMapData(data map[string]interface{}) map[string]int {
```

Comment:
@ishangupta-ds @shobha2626 , this is where generics should be used

Reply:
Should we take it up as an incremental change? Or is it required in this PR?